src/test/resources/disabled.xml

```diff
@@ -0,0 +1,12 @@
+<jenkins.plugins.logstash.LogstashConfiguration plugin="logstash@1.4.0-SNAPSHOT">
+  <logstashIndexer class="jenkins.plugins.logstash.configuration.RabbitMq">
+    <host>localhost</host>
+    <port>5672</port>
+    <queue>logstash</queue>
+    <username></username>
+    <password>{AQAAABAAAAAQ5wlElDfWGri9VuaMh0MCBZWs1fjL31zvnxrkszfW5pA=}</password>
+  </logstashIndexer>
+  <enabled>false</enabled>
```

Comment:
I think this tag would not be present in an actual migration from 1.4 (seeing that you are adding the field now)?

Reply:
This file is not used for testing migration.

Reply:
OK then, `plugin="logstash@1.4.0-SNAPSHOT"` confused me